docs/_layouts/documentation.html

```diff
@@ -27,5 +27,6 @@
   </div>
 </div>
 
+<style>body{overflow: hidden;}</style>
```

Comment:
Well I've checked the pull request that remove the bottom left part of the menu, and it seems that it was generating the scroll on the body. I'll remove this inline for the time being and we'll check again when everything is merged.